### DISCUSSION TITLE
Custom font with `lit`

### DISCUSSION THREAD

**Discussion by 0xbuidler** [lit/lit#4634] [General]
How to add a custom font with `lit` and how can I ensure it is included in the build process ?

**Comment by sorvell** [comment]
Fonts must be loaded in the global scope so this can't be fully encapsulated inside an element. That said, it's possible to have an element inject a font using the [technique shown here](https://lit.dev/playground/#gist=d9a17f54735c9a884fccaa1f49c3386a) (just make sure to put the sheet in the document). If you do it this way, the code should be included with your element so as long as you're building that it'll work. Hope that helps.
